renku_notebooks/api/notebooks.py

```diff
@@ -295,11 +295,74 @@ def launch_notebook(
     manifest = server.start()
 
     current_app.logger.debug(f"Server {server.server_name} has been started")
-    for autosave in user.get_autosaves(server.gl_project.path_with_namespace):
-        autosave.cleanup(server.commit_sha)
+
+    # TODO: Clear persistent sessions for all branch/commits in the project
+
     return NotebookResponse().dump(UserServerManifest(manifest)), 201
 
 
+@bp.route("servers/<server_name>", methods=["PATCH"])
+@use_args(
+    {"server_name": fields.Str(required=True)}, location="view_args", as_kwargs=True
+)
+@use_args(PatchServerRequest(), location="json", as_kwargs=True)
+@authenticated
+def hibernate_or_resume_server(user, server_name, state):
```

Comment:
```suggestion
def patch_server(user, server_name, state):
```

In the future this will grow to do more changes not just hibernate and resume.